jwst/pipeline/calwebb_guider.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+from ..stpipe import Pipeline
+from .. import datamodels
+import os
+
+# step imports
+from ..dq_init import dq_init_step
+from ..flatfield import flat_field_step
+from ..guider_cds import guider_cds_step
+
+__version__ = "7.0.0"
+
+# Define logging
+import logging
+log = logging.getLogger()
+log.setLevel(logging.DEBUG)
+
+class GuiderPipeline(Pipeline):
```

Comment:
Ah, here you are using "GuiderPipeline." Good. So the .cfg file just needs to be updated to be consistent.